src/jit/lsra.cpp

```diff
@@ -3647,49 +3497,63 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
     // gtRsvdRegs register mask. Clear it out.
     tree->gtRsvdRegs = RBM_NONE;
 
-    TreeNodeInfo info = tree->gtLsraInfo;
-    assert(info.IsValid(this));
-    int consume = info.srcCount;
-    int produce = info.dstCount;
+#ifdef DEBUG
+    if (VERBOSE)
+    {
+        dumpOperandToLocationInfoMap();
+        compiler->gtDispTree(tree, nullptr, nullptr, true);
+    }
+#endif // DEBUG
+
+    // If the node produces a value that will be consumed by a parent node, its TreeNodeInfo will
+    // be allocated in the LocationInfoListNode. Otherwise, we'll just use a local value that will
+    // be thrown away when we're done.
+    LocationInfoListNode* locationInfo = nullptr;
+    TreeNodeInfo          tempInfo;
+    TreeNodeInfo*         info = nullptr;
+    // This
```

Comment:
Strange comment

Reply:
Hmm - I'll have to figure out what I was going to say when I apparently got distracted!